sympy/vector/integrals.py

```diff
@@ -0,0 +1,79 @@
+from sympy.core.basic import Basic
+from sympy.vector import CoordSys3D, Vector
+from sympy.vector.operators import _get_coord_sys_from_expr
+from sympy.integrals import Integral, integrate
+from sympy.core.function import diff
+
+class ParametricIntegral(Basic):
+    """
+    Represents integral of a scalar or vector field
+    over a Parametric Region
+
+    Examples
+    ========
+
+    >>> from sympy.vector import CoordSys3D, ParametricRegion
+    >>> from sympy.abc import t
+
+    >>> C = CoordSys3D('C)
+    >>> curve = ParametricRegion(t, (3*t - 2, t + 1), {t: (1, 2)})
+
+    >>> ParametricIntegral(C.x, curve)
+    0
+    >>> semisphere = ParametricRegion((phi, theta), (2*sin(phi)*cos(theta), 2*sin(phi)*sin(theta), 2*cos(phi))
+                            {theta: (0, 2*pi), phi: (0, pi/2)})
+    >>> ParametricIntegral(C.z, semisphere)
+    8*pi
+
+    >>> ParametricIntegral(C.j - C.k, ParametricRegion((r, theta), (r*cos(theta), r*sin(theta))))
+    ParametricIntegral(C.j - C.k, ParametricRegion((r, theta), (r*cos(theta), r*sin(theta))))
+
+    """
+    
+    def __new__(cls, field, parametricregion):
+        obj = super().__new__(cls, field, parametricregion)
+        return obj
```

Comment:
I am not sure how should I make this a subclass of Integral.

Reply:
No need to inherit any classes, unless you really want to reuse some code of the upper class.